sandbox-server/app.js

```diff
@@ -51,4 +51,10 @@ app.get('/somefile', (request, response) => {
   }
 })
 
+app.get('/old-page', (request, response) => {
+  response.status(301)
+  response.setHeader( 'Location', 'http://localhost:3000/newpage')
+  response.send()
```

Comment:
this is a very manual way of doing `response.redirect('/newpage')`.

Also you don't want to inline `http://localhost:3000` because it will have to change if / when your app is hosted somewhere else. Use relative paths.